CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 
 Unreleased
 ==========
+* fix: Adding objects to a collection should be allowed unless locked by as user
```

Comment:
This is not clear to me. Should it be something more like:
"Adding child objects to a collection should be allowed unless locked by another user"?